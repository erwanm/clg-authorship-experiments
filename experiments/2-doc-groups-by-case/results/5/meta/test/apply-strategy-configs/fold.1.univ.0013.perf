0.346094	0.586600	0.590000
