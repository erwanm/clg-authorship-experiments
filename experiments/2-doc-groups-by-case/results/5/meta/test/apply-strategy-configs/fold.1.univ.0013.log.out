apply-config.sh: applying config '/tmp/expe2/5/process/selected-meta-configs/fold.1.0061.model/strategy-configs/fold.1.univ.0013' to cases '/home/moreaue/dump/expe2//results/5/meta/test/apply-strategy-configs/only-cases.tmp.w7B2lHTw'; results written to '/home/moreaue/dump/expe2//results/5/meta/test/apply-strategy-configs/fold.1.univ.0013'
apply-config.sh: done.
